include/nlohmann/ordered_map.hpp

```diff
@@ -45,7 +54,14 @@ struct ordered_map : Container
         {
             if (it->first == key)
             {
-                Container::erase(it);
+                // Since we cannot move const Keys, re-construct them in place
+                for (auto next = it; ++next != this->end(); ++it)
+                {
+                    // *it = std::move(*next); // deleted
```

Comment:
Do we need this commented code?

Reply:
Not really, the comment above should be enough.

Reply:
Can you remove it, please?

Reply:
Yes and I will try to address your comments today...

Reply:
The comment `// *it = std::move(*next); // deleted` is still there. Please remove.